zk_toolbox/crates/zk_inception/src/commands/chain/common.rs

```diff
@@ -111,7 +111,7 @@ pub async fn mint_base_token(
         let amount = AMOUNT_FOR_DISTRIBUTION_TO_WALLETS * base_token.nominator as u128
             / base_token.denominator as u128;
         common::ethereum::mint_token(
-            wallets.operator,
+            wallets.governor,
```

Comment:
Shouldnt this be `wallets.deployer` as `mint_token` is minting tokens to `wallets.governor` ?

Reply:
On second thought, it makes for sense for `governor` to be able to `mint` tokens rather then the `deployer`. Excuse my original comment.